recipes/deltalake/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "deltalake" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deltalake-{{ version }}.tar.gz
+  sha256: b8793a1c7a1219c8935ed9926a6870c9341597396c0f50bf6ff98c323d06ee0f
+
+build:
+  skip: true  # [py<37]
```

Comment:
FWIW conda-forge is [Python 3.8+]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/9f100b9537541429e055de9b16aaa308d1d6264c/recipe/conda_build_config.yaml#L643-L647 ). So this is probably not needed

```suggestion
```